src/docs/content/examples/alert/notifications-default.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div>
+    <div class="playground">
+      <fd-field-set>
+        <fd-form-item>
+          <fd-form-label>Notification Title</fd-form-label>
+          <fd-input v-model="title" placeholder="Title" />
+        </fd-form-item>
+        <fd-form-item>
+          <fd-form-label>Notification Content</fd-form-label>
+          <fd-text-area v-model="content" />
+        </fd-form-item>
+        <fd-form-item>
+          <fd-form-label>Timeout (ms)</fd-form-label>
+          <fd-input :disabled="permanent" type="number" v-model="timeout" />
+        </fd-form-item>
+        <fd-form-group>
+          <fd-form-item-toggle label="Permanent">
+            <fd-toggle v-model="permanent" />
+          </fd-form-item-toggle>
+          <fd-form-item-toggle label="Dismissible">
+            <fd-toggle v-model="dismissible" />
+          </fd-form-item-toggle>
+        </fd-form-group>
+        <br /><br />
+        <fd-form-item>
+          <fd-form-label>Notification Type</fd-form-label>
+          <fd-select v-model="type">
+            <option v-for="t in types" :key="t">{{ t }}</option>
+          </fd-select>
+        </fd-form-item>
+      </fd-field-set>
+      <fd-button @click="notify" styling="emphasized" type="positive"
+        >Create Notification
+      </fd-button>
+    </div>
+    <fd-notifications></fd-notifications>
+  </div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      title: "",
+      timeout: 2000,
+      permanent: false,
+      dismissible: true,
+      content: "Watashi ga kita!",
```

Comment:
![I-SEE-WHAT-YOU-DID-THERE_HOLYSHIT](https://user-images.githubusercontent.com/153225/64003153-f4951300-cb0b-11e9-90b1-96bcb624d2b0.jpg)